auth_ldaps/models/res_company_ldap.py

```diff
@@ -0,0 +1,27 @@
+# © Creu Blanca
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import ldap
+from odoo import api, fields, models
+
+
+class CompanyLDAP(models.Model):
+    _inherit = 'res.company.ldap'
+
+    is_ssl = fields.Boolean(string='Use LDAPS', default=False)
```

Comment:
This might sound like a very stupid question, but AFAIK all SSL protocols were declared as insecure nowadays. Does this work for TLS too?